apps/hatchet-worker-general/src/index.ts

```diff
@@ -107,6 +109,14 @@ async function main() {
 
   const pubSub = createPubSub({ eventTarget })
 
+  const auditClient = new AuditClient()
+
+  auditClient.log({
```

Comment:
these are not working so well yet and we would probably need some unique identifier per worker/similar to identify them. but I think it might still be a good idea to track the "uptime" or crashes in the backend in the audit log? just for generally critical components though..